README.yaml

```diff
@@ -420,52 +460,60 @@ usage: |-
       "Attributes" = "fire-water-earth-air"
       "City" = "Dublin"
       "Environment" = "Private"
-      "Name" = "winstonchurchroom-uat-build-fire-water-earth-air"
+      "Name" = "winstonchurchroom-hrh-uat-build-fire-water-earth-air"
       "Namespace" = "cloudposse"
       "Stage" = "build"
+      "Tenant" = "hrh"
     }
+    "tenant" = "hrh"
   }
-  label1_tags = {
+  label1_tags = tomap({
     "Attributes" = "fire-water-earth-air"
     "City" = "Dublin"
     "Environment" = "Private"
-    "Name" = "winstonchurchroom-uat-build-fire-water-earth-air"
+    "Name" = "winstonchurchroom-hrh-uat-build-fire-water-earth-air"
     "Namespace" = "cloudposse"
     "Stage" = "build"
-  }
+    "Tenant" = "hrh"
+  })
   label2 = {
-    "attributes" = [
+    "attributes" = tolist([
       "fire",
       "water",
       "earth",
       "air",
-    ]
+    ])
     "delimiter" = "+"
     "id" = "charlie+uat+test+fire+water+earth+air"
     "name" = "charlie"
     "namespace" = "cloudposse"
     "stage" = "test"
+    "tenant" = ""
   }
   label2_context = {
     "additional_tag_map" = {
       "additional_tag" = "yes"
       "propagate_at_launch" = "true"
     }
-    "attributes" = [
+    "attributes" = tolist([
       "fire",
       "water",
       "earth",
       "air",
-    ]
+    ])
     "delimiter" = "+"
     "enabled" = true
     "environment" = "UAT"
-    "label_order" = [
+    "id_length_limit" = tonumber(null)
```

Comment:
why use the conversions instead of just `null`?

Reply:
I just copied the Terraform output because the previous README had errors and I did not want to re-introduce errors by modifying the output.